Parsers/ASimWebSession/Test/Citrix_NetScaler_vimWebSession_SchemaTest.csv

```diff
@@ -0,0 +1,124 @@
+"(2) Info: Missing optional field [FileSHA256]"
+"(2) Info: Missing optional field [FileSHA512]"
+"(2) Info: Missing optional field [FileSize]"
+"(2) Info: Missing optional field [HttpContentFormat]"
+"(2) Info: Missing optional field [HttpContentType]"
+"(2) Info: Missing optional field [HttpCookie]"
+"(2) Info: Missing optional field [HttpHost]"
+"(2) Info: Missing optional field [HttpIsProxied]"
+"(2) Info: Missing optional field [HttpReferrer]"
+"(2) Info: Missing optional field [HttpRequestBodyBytes]"
+"(2) Info: Missing optional field [HttpRequestCacheControl]"
+"(2) Info: Missing optional field [HttpRequestHeaderCount]"
+"(2) Info: Missing optional field [HttpRequestTime]"
+"(2) Info: Missing optional field [HttpRequestXff]"
+"(2) Info: Missing optional field [HttpResponseBodyBytes]"
+"(2) Info: Missing optional field [HttpResponseCacheControl]"
+"(2) Info: Missing optional field [HttpResponseExpires]"
+"(2) Info: Missing optional field [HttpResponseHeaderCount]"
+"(2) Info: Missing optional field [HttpResponseTime]"
+"(2) Info: Missing optional field [HttpUserAgent]"
+"(2) Info: Missing optional field [HttpVersion]"
+"(2) Info: Missing optional field [NetworkApplicationProtocol]"
+"(2) Info: Missing optional field [NetworkBytes]"
+"(2) Info: Missing optional field [NetworkConnectionHistory]"
+"(2) Info: Missing optional field [NetworkDirection]"
+"(2) Info: Missing optional field [NetworkDuration]"
+"(2) Info: Missing optional field [NetworkIcmpCode]"
+"(2) Info: Missing optional field [NetworkIcmpType]"
+"(2) Info: Missing optional field [NetworkPackets]"
+"(2) Info: Missing optional field [NetworkProtocolVersion]"
+"(2) Info: Missing optional field [NetworkProtocol]"
+"(2) Info: Missing optional field [RuleName]"
+"(2) Info: Missing optional field [RuleNumber]"
+"(2) Info: Missing optional field [Rule]"
+"(2) Info: Missing optional field [SrcAppId]"
+"(2) Info: Missing optional field [SrcAppName]"
+"(2) Info: Missing optional field [SrcAppType]"
+"(2) Info: Missing optional field [SrcBytes]"
+"(2) Info: Missing optional field [SrcDescription]"
+"(2) Info: Missing optional field [SrcDeviceType]"
+"(2) Info: Missing optional field [SrcDvcId]"
+"(2) Info: Missing optional field [SrcFQDN]"
+"(2) Info: Missing optional field [SrcGeoCity]"
+"(2) Info: Missing optional field [SrcGeoLatitude]"
+"(2) Info: Missing optional field [SrcGeoLongitude]"
+"(2) Info: Missing optional field [SrcGeoRegion]"
+"(2) Info: Missing optional field [SrcInterfaceGuid]"
+"(2) Info: Missing optional field [SrcInterfaceName]"
+"(2) Info: Missing optional field [SrcMacAddr]"
+"(2) Info: Missing optional field [SrcNatIpAddr]"
+"(2) Info: Missing optional field [SrcNatPortNumber]"
+"(2) Info: Missing optional field [SrcOriginalUserType]"
+"(2) Info: Missing optional field [SrcPackets]"
+"(2) Info: Missing optional field [SrcUserId]"
+"(2) Info: Missing optional field [SrcUserType]"
+"(2) Info: Missing optional field [SrcUsername]"
```

Comment:
Pls check for this field too: SrcUsername

Reply:
No such field is available which we can map with the SrcUsername.